flow-server/src/test/java/com/vaadin/flow/server/connect/ExplicitNullableTypeCheckerTest.java

```diff
@@ -423,6 +442,15 @@ public String stringNullable() {
         return "";
     }
 
+    /**
+     * Method for testing
+     */
+    @Id
+    @Nullable
+    public Long methodWithIdAnnotation(){
+        return null;
+    }
```

Comment:
Let’s remove `@Nullable` from here. Otherwise, even if `@Id` is not implemented properly in the checker, the test might still pass because of `@Nullable` here.
```suggestion
    public Long methodWithIdAnnotation() {
        return 1L;
    }
```